### DISCUSSION TITLE
How to Remove: the `Web console: https://unifisrv:9090/ or https://192.168.0.200:9090/` from the MOTD

### DISCUSSION THREAD

**Discussion by SlowFox72** [cockpit-project/cockpit#20493] [Q&A] [answered]
Is it possible to remove the `Web console: https://unifisrv:9090/ or https://192.168.0.200:9090/` message from the MOTD

### Version of Cockpit

316-1~bpo22.04.1

### Where is the problem in Cockpit?

None

### Server operating system

Ubuntu

### Server operating system version

22.04.4 LTS

### What browsers are you using?

Brave

**Comment by SlowFox72** [comment]
What I have done is the following:

`sudo mv /run/cockpit/active.motd  /run/cockpit/active.motd.org`
and I emptied the `/etc/issue.d/cockpit.issue` file (I left the file but emptied the contents)
and I emptied the `/etc/motd/cockpit` (I left the file but I emptied the contents)

Are there more elegant ways to remove the MOTD from cockpit?

**Comment by martinpitt** [comment, marked as answer]
No need to touch `/run` - that's not configuration, and will be overwritten at next startup. Removing the files in /etc is indeed the correct and supported way to do this.

**Comment by SlowFox72** [comment]
@martinpitt 

Thanks for the info and confirmation.
